region/crateria/east.json

```diff
@@ -534,13 +666,19 @@
           "to": [
             {
               "id": 12,
-              "requires": [
-                {"or": [
-                  "canPassBombPassages",
-                  "canUseSpringBall"
-                ]}
-              ],
-              "note": "The shot blocks can be destroyed without using bombs"
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "canPassBombPassages",
+                      "canUseSpringBall"
```

Comment:
you can bug boost through here too.  Only get 1 shot and you have to be quick.

Reply:
Thanks, I hate it.
I think I'm going to have to required mockball on this one.